packages/devextreme/js/__internal/grids/new/card_view/content_view/content/card/card.image.tsx

```diff
@@ -0,0 +1,29 @@
+import { PureComponent } from '@ts/grids/new/grid_core/core/pure_component';
+import type { InfernoNode } from 'inferno';
+
+const CLASSES = {
+  cover: 'dx-card-cover',
+};
+
+export interface ImageProps {
+  src?: string;
```

Comment:
`src` shouldn't be optional. There's no point in creating Cover if there's no src. But it can be optional in Card